book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Hi. as part of the requirement, you are supposed to create a has-many relationship side of Book and Rental (a book has many rentals). So, you are supposed to create a method for adding rentals in the book class and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

Reply:
Thank you @m05e5 , let me add that part!